packages/qemu.rb

```diff
@@ -31,14 +33,18 @@ class Qemu < Package
   depends_on 'hicolor_icon_theme'
 
   def self.build
-    system './configure',
-           "--prefix=#{CREW_PREFIX}",
-           "--libdir=#{CREW_LIB_PREFIX}",
-           "--disable-stack-protector"
-    system 'make'
+    FileUtils.mkdir_p 'build'
+    Dir.chdir 'build' do
+      system "#{CREW_ENV_OPTIONS} ../configure #{CREW_OPTIONS.sub(/--target.*/, '')} \
```

Comment:
Use `meson`?

Reply:
I don't think you understand what it means to ask @satmandu to compile this all over again. :)